### PR TITLE
Catelog of my things project

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR we created the following:

A console app that will help you to keep a record of different types of things you won: books, music albums, movies, and games. Everything will be based on the UML class diagram given. The data will be stored in JSON files and preparation of a database with tables structure analogical to your program's class structure.

### CODE REVIEW

File: src/modules/create_game.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+require_relative '../author'
+require_relative '../genre'
+require_relative '../label'
+require_relative '../game'
+require_relative 'main_methods'
+require_relative 'create_music'
+require_relative 'create_book'
+require_relative 'create_movie'
+
+module CreateGame
```
Comment (omar-labana) [inline src/modules/create_game.rb:10]: - [x] Creating a game crashes the program. I'd suggest using begin-rescue to catch these types of errors  
image

File: src/book.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+class Book < Item
```
Comment (omar-labana) [inline src/book.rb:2]: - [x] All public methods in all classes should have unit tests. please add tests for the rest of the classes.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi team, 

Excellent work so far! 👌
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [x] JWT token is missing in the header, please attach it with devise_jwt.
- [x] Make sure the JWT blacklist should be added.  

## Optional Suggestions

- You can Leverage Devise Gem and customise every step of the Authentication.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi Team, 

Good job on implementing the suggested changes. Now the App is working fine and you have added more tests. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**